### ISSUE TITLE
Block inserter lacking a clear close button

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#47016]
As reported in https://github.com/WordPress/gutenberg/issues/46939#issuecomment-1374652632 the block inserter is lacking a clear close button. This makes it very difficult for users of assistive technology to exist the popover.

A clear button should be provided which closes the inserter and returns focus to the last focused element.

**Comment by draganescu** [comment]
This seems to be a feature request for the quick inserter.

**Comment by alexstine** [comment]
Yes but a must none the less. Not sure how I missed this elsewhere. Maybe I just never used it anywhere else before now. Escape is not a given sadly so having a close button makes it clear.

**Comment by carolinan** [comment]
I think this needs some design guidance before it can be implemented.

**Comment by jasmussen** [comment]
Related to [a conversation here](https://github.com/WordPress/gutenberg/issues/47009#issuecomment-1381769347), I'd like to hear thoughts on having it be an on-focus close button that appears, for example, at the end of the list of tabbables.

**Comment by ciampo** [comment]
As @jasmussen suggests, could we add a visually hidden button that can be focused via keyboard navigation? The button could become visually visible when focused, similarly to the ["skip to content" links](https://www.w3.org/TR/WCAG20-TECHS/G1.html) that are usually added at the top of a web page.

**Comment by getdave** [comment]
I implemented a proof of concept in https://github.com/WordPress/gutenberg/pull/47327

**Comment by getdave** [comment]
<img width="791" alt="Screen Shot 2023-01-30 at 10 18 14 (1)" src="https://user-images.githubusercontent.com/444434/215450049-4b483e7f-65a8-4506-976a-6b1db6b384d0.png">

Here's an example of where we already include a clear Close button in the UI. I wonder whether would could borrow the same pattern?

**Comment by karmatosed** [comment]
I think having the design feedback label makes sense. I am not sure this also needs design, so for now going to focus back on that label.

**Comment by jasmussen** [comment]
This appears resolved. The inserter now uses a shared TabbedSidebar component (packages/block-editor/src/components/tabbed-sidebar) that renders a visible, labeled close button, added in #62343 and refined in #66756 (both merged). Closing as fixed. Please reopen with a repro if this was in error.
